### PR TITLE
Kaspersky.Lab version 1.0

### PR DESCRIPTION
**Pull request by RanierBrolessi**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90361)

### CODE REVIEW

**Comment by RanierBrolessi** [comment]
@microsoft-github-policy-service agree [company="{}"]

**Comment by RanierBrolessi** [comment]
> @RanierBrolessi please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement

@microsoft-github-policy-service agree [company="{}"]

**Comment by RanierBrolessi** [comment]
@microsoft-github-policy-service agree

File: manifests/k/Kaspersky/Lab/1.0/Kaspersky.Lab.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Kaspersky.Lab
+PackageVersion: 1.0
+Installers:
+- Architecture: x64
+  InstallerType: portable
+  InstallerUrl: https://download1506.mediafire.com/wad68t3os59g/xsgspau91udkrkw/Kaspersky_Endpoint_Security_11.6_com_Agent.exe
```
Comment (russellbanks) [inline manifests/k/Kaspersky/Lab/1.0/Kaspersky.Lab.installer.yaml:9]: A mediafire link isn't suitable for an InstallerUrl. Where did you get this from?

**Review by russellbanks — CHANGES_REQUESTED** [review]

**Review by russellbanks — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/31783#pullrequestreview-788226536
The software needs to come from the ISV's server.

**Pull request closed without merging**